flask_sqlalchemy/cli.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+    flask_sqlalchemy.cli
+    ~~~~~~~~~~~~~~~~~~~~
+
+    Command Line Interface for managing the database.
+
+    :copyright: (c) 2017 by David Baumgold.
+    :license: MIT, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+from functools import wraps
+
+import click
+from flask import current_app
+from werkzeug.local import LocalProxy
+
+try:
+    from flask.cli import with_appcontext
+except ImportError:
+    from flask_cli import with_appcontext
+
+state = LocalProxy(lambda: current_app.extensions['sqlalchemy'])
+
+
+def commit(fn):
+    """Decorator to commit changes after the command is run."""
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        fn(*args, **kwargs)
+        state.db.session.commit()
+    return wrapper
+
+
+@click.group()
+def db():
+    """Manages the database."""
+
+
+@db.command('create')
+@click.option('--bind', default='__all__')
+@with_appcontext
+@commit
+def db_create(bind):
+    """Creates database tables."""
+    state.db.create_all(bind=bind)
+    click.secho('Database created successfully.', fg='green')
+
+
+@db.command('drop')
+@click.option('--bind', default='__all__')
+@click.confirmation_option(prompt=(
+    "This will destroy all the data in your database. "
```

Comment:
I'm talking about a testing flow that isn't necessarily in Python. For example, a Jenkins worker that sets up the codebase and then runs Selenium tests. That sort of scenario would generally be scripted in bash, not in Python.

I hadn't thought about `FLASK_DEBUG=1` -- but if anything, that makes me even less keen to make this change. The confirmation option can be bypassed by running `flask db drop --yes` -- what makes you think it's any more difficult to just run `FLASK_DEBUG=1 flask db drop --yes` instead?

I'm all for avoiding footguns, but this smacks of child-proofing that is too easily disabled to be useful. I believe we should make sure that people are aware of the risks of what they're doing (hence the confirmation option), and then trust that they're smart enough to do the right thing.